WMF Framework/WMFExploreFeedContentController.m

```diff
@@ -72,7 +72,7 @@ - (void)setOperationQueue:(NSOperationQueue *)operationQueue {
 - (void)setDataStore:(MWKDataStore *)dataStore {
     _dataStore = dataStore;
     self.exploreFeedPreferencesUpdateCoordinator = [[ExploreFeedPreferencesUpdateCoordinator alloc] initWithFeedContentController:self];
-    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(viewContextDidSave:) name:NSManagedObjectContextDidSaveNotification object:self.dataStore.viewContext];
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(feedImportContextDidSave:) name:WMFFeedImportContextDidSave object:nil];
```

Comment:
Since the view context can also write changes to the PSC, it should be observed as well